robottelo/ui/sync.py

```diff
@@ -129,7 +129,11 @@ def enable_rh_repos(self, repos_tree):
         :return: None.
 
         """
-        strategy, value = locators['rh.prd_expander']
+        if ostree:
```

Comment:
Yeah.. Agreed. Current implementations is around enabling on `rpm` based repos and that too only from first tab. The one I added is for ostree. It depends on what all repos we want to sync. For example: If later, someone wants to sync `kickstart trees` then those repos would only be available from another tab. So there are 8 tabs on `RedHat repositories` page. I'll see if I can make it generic.